frontend/src/components/Board/BoardShowcase.module.scss

```diff
@@ -0,0 +1,209 @@
+$alto: #cfcfcf;
+$alabaster: #fafafa;
+$azure-radiance: #0c9bef;
+$black: #000;
+$cod-gray: #1b1b1b;
+$champagne: #faebcc;
+$crete: #52842f;
+$dove-gray: #696969;
+$dusty-gray: #969696;
+$emerald: #5aca5e;
+$fruit-salad: #4caf50;
+$gallery: #eee;
+$gray: #808080;
+$malachite: #37e039;
+$mine-shaft: #373737;
+$new-york-pink: #d67b7b;
+$pearl-lusta: #fcf8e3;
+$potters-clay: #8a6d3b;
+$puce: #d29191;
+$scorpion: #585858;
+$silver: #c5c5c5;
+$silver-chalice: #a9a9a9;
+$white: #fff;
+$lightlightgray: #ececec;
+
+//these are also defined in media_queries.js and need to be adjusted there and here
+@mixin breakpoint($point) {
+  @if $point == small_width {
+    @media (max-width: 1200px) {
+      @content;
+    }
+  } @else if $point == semantic_container {
+    @media (max-width: 1199px) {
+      @content;
+    }
+  } @else if $point == mobile-m {
+    @media (max-width: 499px) {
+      @content;
+    }
+  } @else if $point == mobile_menu {
+    @media (max-width: 590px) {
+      @content;
+    }
+  }
+}
+
```

Comment:
Why are these repeated everywhere? Can we not `@import` them?

Reply:
Yes, all the styles need proper reorganization.

Reply:
Can you do that on this PR please.

Reply:
Resolved via e2043bc.